dex/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     compile "com.google.code.findbugs:jsr305:${versions.jsr305}"
     compile "com.google.guava:guava:${versions.guava}"
     compile "org.apache.lucene:lucene-core:${versions.lucene}"
+    compile 'com.carrotsearch:hppc:0.7.1'
```

Comment:
Can we add a versions entry since this is now used in multiple places?

Reply:
sure